main.py

```diff
@@ -40,5 +41,6 @@ def root():
     # the "static" directory. See:
     # http://flask.pocoo.org/docs/1.0/quickstart/#static-files. Once deployed,
     # App Engine itself will serve those files as configured in app.yaml.
+    print(create_user())
```

Comment:
I assume this is for testing and to show us how it works. But maybe can remove cos we won't want to put this here, and it might cause confusion down the road.